documentation/tarron_notes.md

```diff
@@ -1,5 +1,15 @@
 # Tarron's Notes
 
+## Questions
+
+- [ ] What does this notation `(rootQuery, { planId }, context)` mean in lambda function declaration
+- [ ] Add Stripe CustomerID on space center?
```

Comment:
Rather, the space center GraphQL type will have a stripe object on it of type StripeCustomer which provides the necessary information. There will be only one of them.